lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -664,7 +666,10 @@ private function _parseOptions(SimpleXMLElement $options)
             $attr = $option->attributes();
 
             if (isset($attr->name)) {
-                $array[(string) $attr->name] = $value;
+                $attrName = (string) $attr->name;
```

Comment:
Please don't abbreviate. I know the previous code did, but we should really call it `$attributeName` or such